hadoop-common-project/hadoop-common/src/test/java/org/apache/hadoop/io/compress/CompressDecompressTester.java

```diff
@@ -495,19 +479,16 @@ public String getName() {
     Compressor compressor = pair.compressor;
 
     if (compressor.getClass().isAssignableFrom(Lz4Compressor.class)
-            && (NativeCodeLoader.isNativeCodeLoaded()))
```

Comment:
nit: unrelated changes :)

Reply:
Oh, this is from @dbtsai's original change. I think adding curly brackets is better? I can revert this if you think it is necessary.

Reply:
Yeah usually it's not recommended to include unrelated changes in Hadoop patch, we may add another refactoring PR later if this is absolutely necessary.

Reply:
Ok. Reverted the change.